core-bundle/src/Monolog/ContaoTableProcessor.php

```diff
@@ -46,7 +46,7 @@ public function __invoke(array $record): array
 
         $context = $record['context']['contao'];
         $request = $this->requestStack->getCurrentRequest();
-        $level = $record['level'] ?? 0;
+        $level = $record['level'];
```

Comment:
Why?

Reply:
See here: https://github.com/contao/contao/pull/3664#issuecomment-965826213

Reply:
> How can we be sure? 

The array structure comes from monolog and they are annotating it; see https://github.com/Seldaek/monolog/blob/f19a2ae873d42f4494f3fb29de9dadb91a3f7601/src/Monolog/Logger.php#L30-L32.

----

Basically the `ProcessorInterface` says it is a `Record`.

The `Logger` defines a `Record` looks like this:
```
array{message: string, context: mixed[], level: Level, level_name: LevelName, channel: string, datetime: \DateTimeImmutable, extra: mixed[]}
```

And also that a `Level` looks like this:
```
Logger::DEBUG|Logger::INFO|Logger::NOTICE|Logger::WARNING|Logger::ERROR|Logger::CRITICAL|Logger::ALERT|Logger::EMERGENCY
```

These constants evaluate to `100|200|250|300|400|500`.